sdcm/cluster.py

```diff
@@ -1732,6 +1732,9 @@ def install_package(self, package_name: str) -> None:
             pkg_cmd = 'zypper'
         else:
             pkg_cmd = 'apt-get'
+            if package_name == "scylla-manager-agent":
+                # A workaround for: https://github.com/scylladb/scylla-pkg/issues/2578
+                self.remoter.sudo('apt --fix-broken install -y')
```

Comment:
here you could, instead of running it, just make:
```
pkg_cmd = 'apt --fix-broken'
```

and then having it run on next line (otherwise, you will have it running twice)

Reply:
it is actually two different commands , following the previous apt-get response recommendation.

Reply:
Please take this out to a separate commit.